### PR TITLE
docs: beta vs stable package install instructions

### PR DESCRIPTION
**Pull request by mmkal**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,26 @@
-# Umzug [![Build Status](https://travis-ci.org/sequelize/umzug.svg?branch=master)](https://travis-ci.org/sequelize/umzug) [![npm version](https://badgen.net/npm/v/umzug)](https://www.npmjs.com/package/umzug)
+# Umzug
+
+[![Build Status](https://img.shields.io/github/workflow/status/sequelize/umzug/CI/master)](https://github.com/sequelize/umzug/actions?query=workflow%3ACI)
+[![npm (stable)](https://badgen.net/npm/v/umzug)](https://www.npmjs.com/package/umzug)
+[![npm (beta)](https://img.shields.io/npm/v/umzug/beta?color=cyan)](https://www.npmjs.com/package/umzug/v/beta)
```
Comment (papb) [inline README.md:5]:
```suggestion
[![npm (beta)](https://badgen.net/npm/v/umzug/beta)](https://www.npmjs.com/package/umzug/v/beta)
```

File: README.md [hunk 2]
```diff
@@ -1,9 +1,26 @@
...
+[![npm (downloads)](https://img.shields.io/npm/dm/umzug)](https://npmjs.com/package/umzug)
 
 Umzug is a framework-agnostic migration tool for Node. It provides a clean API for running and rolling back tasks.
 
 _Note: master represents the next major version of umzug - v3 - which is currently in beta. For the stable version, please refer to the [v2.x branch](https://github.com/sequelize/umzug/tree/v2.x)._
 
+To install the beta package:
```
Comment (papb) [inline README.md:12]:
```suggestion
To install the v3-beta package:
```

File: README.md [hunk 3]
```diff
@@ -1,9 +1,26 @@
-# Umzug [![Build Status](https://travis-ci.org/sequelize/umzug.svg?branch=master)](https://travis-ci.org/sequelize/umzug) [![npm version](https://badgen.net/npm/v/umzug)](https://www.npmjs.com/package/umzug)
+# Umzug
+
+[![Build Status](https://img.shields.io/github/workflow/status/sequelize/umzug/CI/master)](https://github.com/sequelize/umzug/actions?query=workflow%3ACI)
+[![npm (stable)](https://badgen.net/npm/v/umzug)](https://www.npmjs.com/package/umzug)
+[![npm (beta)](https://img.shields.io/npm/v/umzug/beta?color=cyan)](https://www.npmjs.com/package/umzug/v/beta)
+[![npm (downloads)](https://img.shields.io/npm/dm/umzug)](https://npmjs.com/package/umzug)
```
Comment (papb) [inline README.md:6]:
```suggestion
[![npm (downloads)](https://badgen.net/npm/dm/umzug)](https://npmjs.com/package/umzug)
```

**Review by papb — CHANGES_REQUESTED** [review]
Also please [remove this](https://github.com/sequelize/umzug/blame/master/README.md#L128-L134) because it got redundant

**Comment by mmkal** [comment]
@papb I kept in the installation instructions in since that's probably where they'll stay once v3 is stable, I wanted to distinguish the options near the top while two options are available for people.

**Pull request merged**